### PR TITLE
Fix: Add AVAIL message to Engine page #958

### LINKED ISSUE #958: [BUG] No "Avail" shows on upper ecam when engine started
**Mod Version**
master

**Describe the bug**
"Avail" Should appear in upper ecam engine indication when N1 more than 18%

**To Reproduce**
1. Start engine
2. After N1 18% no Avail shows

**Expected behavior**
"Avail" appear in upper ecam engine indication when N1 is more than 18%

**Actual behavior**
Nothing appear in upper ecam engine indication

**References**
https://youtu.be/4NAs7DJ42eM?t=116

**Additional context**
Nil

Was this working before/when did the issue start occurring?
No
Is this a problem in the vanilla unmodded game?
Yes

### PR DESCRIPTION
**Pull request by RichardPilbery**
Fixes #958 

## Summary of Changes

During engine start an AVAIL message should appear after N1 exceeds 18.3% and display for 10 seconds.

## Screenshots (if necessary)

**AFTER**

AVAIL-message

## References

[A320neo engine start](https://youtu.be/4NAs7DJ42eM?t=109)

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by Snapmatics** [comment]
LGTM

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 1]
```diff
@@ -1588,10 +1610,22 @@ var A320_Neo_UpperECAM;
...
                 return "REV";
+            } else if (this.timerAvail >= 0) {
+                return "AVAIL";
             } else {
                 return "";
             }
         }
+        getEngineStartStatus() {
+            const engineId = this.index + 1;
+            const value = (SimVar.GetSimVarValue("GENERAL ENG STARTER:" + engineId, "bool") === 1) ? true : false;
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1621]: There's no need for this ternary, the condition is already producing a boolean value.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 2]
```diff
@@ -1588,10 +1610,22 @@ var A320_Neo_UpperECAM;
...
+            return value;
+        }
+        getIgnitionStatus() {
+            const engineId = this.index + 1;
+            const value = (SimVar.GetSimVarValue("TURB ENG IS IGNITING:" + engineId, "bool") === 1) ? true : false;
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1626]: Same here.

**Comment by danbw9** [comment]
LGTM. I understand it's only a draft, but I tested it and works as described and like the real aircraft.

**Review by Benjozork — APPROVED** [review]

**Comment by starvingjavi** [comment]
Name: Javi#2557
Date of testing: Current date 22/10/2020
Version of the sim: 1.9.5
PR Tested: #1491 

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Start from C&D
2. Follow start up procedure
3. Check ECAM for AVAIL message

Overall Rating: Good
Conclusions of the testing: Works as intended and like the example video. Now we are only missing the "COOLING" messages

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.css [hunk 3]
```diff
@@ -340,3 +341,20 @@ a320-neo-ecam-gauge #RootSVG #CurrentValueBorder {
 a320-neo-ecam-gauge #RootSVG #GraduationsGroup line.InnerMarker {
     stroke-width: 1px !important;
 }
+
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.css:344]:
```suggestion
```

**Comment by danbw9** [comment]
@starvingjavi the COOLING message is only on PW engines from neo.

**Comment by danbw9** [comment]
@starvingjavi COOLING message and DUAL COOLING only appears on PW version engines of the A320neo.

File: .github/CHANGELOG.md [hunk 4]
```diff
@@ -69,8 +69,10 @@
 1. [MODEL] Spinner spiral now shows at medium blur - @lukecologne (luke)
 1. [MFD] Weather Radar now sweeps at 180 degrees, Correct masking for flight path/constr/waypoints and display elements, disabled WX/TERR ON MD in PLAN - @2hwk (2Cas#1022)
 1. [ECAM] Added ECAM ALL button functionality @MikeKuijper (Mike Kuijper)
-1. [ECAM] Fix thrust lever position indicator on upper ECAM @RichardPilbery (Richard Pilbery | tricky_dicky3571 on discord)
-1. [ECAM] Adjusted Engine gauges so cursor extends beyond radius of gauge @RichardPilbery (Richard Pilbery)
+1. [ECAM] Adjusted Engine gauges so cursor extends beyond radius of gauge @RichardPilbery (Richard Pilbery | tricky_dicky#3571 on discord)
+1. [ECAM] Added AVAIL message on Engine display after engine start @RichardPilbery (Richard Pilbery | tricky_dicky#3571 on discord)
+1. [ECAM] Fix thrust lever position indicator on upper ECAM @RichardPilbery (Richard Pilbery | tricky_dicky#3571 on discord)
+1. [ECAM] Adjusted Engine gauges so cursor extends beyond radius of gauge (Richard Pilbery | tricky_dicky#3571 on discord)
```
Comment (frankebersoll) [inline .github/CHANGELOG.md:75]: Duplicate of line 72

Comment (RichardPilbery) [reply .github/CHANGELOG.md:75]: Well spotted! Will amend

**Comment by Carter633** [comment]
Name: British Avgeek YT -- Carter633 #1436
Date of testing: 23/10/2020
Version of the sim: 1.9.5.0
PR Tested: #1491 

Testing technique: Full Flight from Cold and Dark

Steps made to test the Pull Request: 
1.Spawned Cold and Dark - worked through to push back and start in Vatsim 
2. Checked engine display during start up
3. AVAIL message displayed at the correct moment as described

Overall Rating: Good
Conclusions of the testing: works well and no issues found, nice to have an added touch to start up so great work!

**Pull request merged**